apps/arweave/src/ar_poa.erl

```diff
@@ -79,63 +85,66 @@ validate(Args) ->
 			end
 	end.
 
-%% @doc Validate the TXPath and DataPath for a chunk. This will return the ChunkID but won't
-%% validate that the ChunkID is correct.
-%% 
-%% SPoA: the proof of access
-%% RecallOffset: the absoluteoffset of the recall byte - 
- validate_paths(#poa{} = SPoA, TXRoot, RecallOffset, BlockStartOffset, BlockSize) ->
+chunk_proof(#chunk_metadata{} = ChunkMetadata, AbsoluteOffset) ->
```

Comment:
Should we rename AbsoluteOffset to e.g. SeekByte because AbsoluteOffset is the end offset in other places?

Reply:
Like this?
https://github.com/ArweaveTeam/arweave/pull/765/commits/fa674567a90713969d8eedf163d9edb9b500ef2a